theSrc/scripts/LabeledScatter.js

```diff
@@ -220,21 +222,21 @@ class LabeledScatter {
     }
   }
 
-  getLegendPointsRect (plotly_chart_layout, is_legend_points_to_right_of_plotly_legend, nsewdrag_rect, config) {
-    if (is_legend_points_to_right_of_plotly_legend) {
+  getLegendPointsRect (plotly_chart_layout, is_legend_elements_to_right_of_plotly_legend, nsewdrag_rect, config) {
+    if (is_legend_elements_to_right_of_plotly_legend) {
       return {
         x: nsewdrag_rect.width + this.plotlyLegendOrColorBarWidth(),
```

Comment:
Doesn't this only work if legend.x  = 1? If its larger or smaller it wouldn't work?

Reply:
I actually can't think of a good reason why you'd move your legend far away from your chart, but I think the other suggestion should fix this.

Reply:
Good point, I'll look into defining x based on the right of the plotly legend.